server/routes/shared_api/variable_group.py

```diff
@@ -21,21 +21,19 @@
 bp = Blueprint("variable-group", __name__, url_prefix='/api/variable-group')
 
 
-@bp.route('/info', methods=['GET', 'POST'])
-def get_variable_group_info():
+@bp.route('/info', methods=['POST'])
```

Comment:
To confirm, there are no clients that use GET today?  Can we tell from monitoring dashboard?

Reply:
All client uses POST now: 
https://github.com/search?q=repo%3Adatacommonsorg%2Fwebsite%20%2Fapi%2Fvariable-group%2Finfo&type=code

Reply:
I thought in the past the Stanford team was using it?

Reply:
don't quite remember, maybe it's mixer api? Also external client should not use/rely on the flask endpoints to fetch data..

Reply:
uh oh, I should not review code, I was thinking this was mixer :)